codecov.yml

```diff
@@ -6,6 +6,6 @@ coverage:
   status:
     project:
       operator:
-        target: 58
+        target: 83
```

Comment:
We might keep on 80 only. :thinking:

Reply:
I was thinking of setting it to `auto` instead of keeping an actual value from now on, maybe with a 2% or 4% threshold. This way the target will be the one on the latest commit. Explained [here](https://docs.codecov.io/docs/commit-status).

Say coverage is currently at 83% and we're using a 4% threshold, `auto` will set the target to that and the new PR will pass the check as long as its coverage is > 79%

Wdyt?

Reply:
+1 for using this to auto and maybe setting the threshold to 5%

Reply:
+1 for auto and 5%. I didn't know that we could have this possibility. Go ahead!